extendr-api/src/wrapper/environment.rs

```diff
@@ -108,6 +109,7 @@ impl Environment {
         self
     }
 
+    #[cfg(feature = "non-api")]
     /// Iterate over an environment.
     pub fn iter(&self) -> EnvIter {
         unsafe {
```

Comment:
This is a breaking change. Needs to be in changelog

We `iter()` method for an environment is now non-api feature which means that if anyone was using this, it will now no longer be cran compliant and that they need to add a feature flag.

Reply:
@JosiahParry the changelog doesn't contain this.

Reply:
> ...various `Environment`, `Function`, `Primitive`, and `Promise` methods.

do you want the iter method specifically mentioned here?

Reply:
No, not at all.